VX03-JP/c100205001.lua

```diff
@@ -0,0 +1,102 @@
+--フレイム・ウィングマン－フレイム・シュート
+--scripted by Raye
+local s,id,o=GetID()
+function s.initial_effect(c)
+	--fusion material
+	c:EnableReviveLimit()
+	aux.AddFusionProcFunRep(c,s.ffilter,2,true)
```

Comment:
> **Must be** Fusion Summoned.
```suggestion
	aux.AddFusionProcFunRep(c,s.ffilter,2,true)
	--spsummon condition
	local e4=Effect.CreateEffect(c)
	e4:SetType(EFFECT_TYPE_SINGLE)
	e4:SetProperty(EFFECT_FLAG_CANNOT_DISABLE+EFFECT_FLAG_UNCOPYABLE)
	e4:SetCode(EFFECT_SPSUMMON_CONDITION)
	e4:SetValue(aux.fuslimit)
	c:RegisterEffect(e4)
```
Copied from `Elemental HERO Nova Master`'s script